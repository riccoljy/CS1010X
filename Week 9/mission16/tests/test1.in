2
A1 3
B1 5