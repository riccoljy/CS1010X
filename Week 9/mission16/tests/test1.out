Bus simulator is ready!
There are a total of 2 events.
3: Bus A1 arrives at PGP (START).
5: Bus B1 arrives at KR BUS TERMINAL.
6: Bus A1 arrives at KR MRT.
9: Bus A1 arrives at LT 27.
10: Bus B1 arrives at IT (OPP CLB).
11: Bus A1 arrives at UNIVERSITY HALL.
11: Bus B1 arrives at OPP YIH.
13: Bus A1 arrives at OPP UHC.
13: Bus B1 arrives at UNIVERSITY TOWN.
14: Bus A1 arrives at YIH.
15: Bus B1 arrives at YIH.
16: Bus A1 arrives at CENTRAL LIBRARY.
16: Bus B1 arrives at CENTRAL LIBRARY.
18: Bus A1 arrives at LT 13.
18: Bus B1 arrives at LT 13.
20: Bus B1 arrives at AS 5.
21: Bus B1 arrives at BIZ 2.
23: Bus A1 arrives at AS 5.
29: Bus A1 arrives at COM 2.
32: Bus A1 arrives at BIZ 2.
33: Bus A1 arrives at OPP TCOMS.
34: Bus A1 arrives at PGP (END).
